tests/storage/storage_migration/test_mtc_storage_class_migration.py

```
@@ -0,0 +1,129 @@
+            vms_with_written_file_before_migration=written_file_to_vms_before_migration,
+            target_storage_class=target_storage_class,
+        )
+
+    @pytest.mark.dependency(depends=[f"{TESTS_CLASS_NAME_A_TO_B}::test_vm_storage_class_migration_a_to_b_running_vms"])
+    @pytest.mark.polarion("CNV-11504")
+    def test_migrate_vms_after_storage_migration(self, booted_vms_for_storage_class_migration):
+        vms_failed_migration = {}
+        for vm in booted_vms_for_storage_class_migration:
+            try:
+                migrate_vm_and_verify(vm=vm, check_ssh_connectivity=True)
+            except Exception as migration_exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)